src/components/core/tests/GeoChart.spec.tsx

```diff
@@ -2,19 +2,96 @@
 import * as React from "react";
 import { mount, ReactWrapper } from "enzyme";
 import { testUtils } from "@gooddata/js-utils";
+import { Execution } from "@gooddata/typings";
 
 import { GeoChartInner, IGeoChartInnerProps } from "../GeoChart";
 import { createIntlMock } from "../../visualizations/utils/intlUtils";
 
 const intl = createIntlMock();
 
+function getExecutionResponse(): Execution.IExecutionResponses {
+    const executionResult: Execution.IExecutionResult = {
```

Comment:
If possible, please reuse/update the existing function `getExecutionResult` from 
`stories/data/geoChart.ts`

Reply:
I already draft one here
https://github.com/gooddata/gooddata-react-components/pull/1315/files#diff-81c9a5878679f7dd5e0f74ba1a410ff6R1100